src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -186,23 +208,29 @@ void positiveAnonymiseProsecutorEntity() {
         assertLastModifiedByAndAt(prosecutorEntity, userAccount);
     }
 
-    @Test
-    void positiveAnonymiseTranscriptionCommentEntity() {
+    @ParameterizedTest(name = "Anonymise TranscriptionCommentEntity with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void anonymiseTranscriptionCommentEntity(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated